lib/modules/manager/tekton/extract.spec.ts

```diff
@@ -9,7 +9,7 @@ describe('modules/manager/tekton/extract', () => {
         'test-file.yaml'
       );
       expect(result).toMatchSnapshot();
-      expect(result?.deps).toHaveLength(20);
+      expect(result?.deps).toHaveLength(39);
```

Comment:
do we really need so many new test deps?

Reply:
IMO this is the only way to ensure all the possible cases are covered. I can remove some, but that would make the tests weaker. Please, let me know how you'd like me to proceed.